RetPageOriginDockerfile

```diff
@@ -0,0 +1,30 @@
+###
+# this dockerfile produces image/container that serves customly packaged hubs and admin static files
+# the result container should serve reticulum as "hubs_page_origin" and "admin_page_origin" on (path) "/hubs/pages"
+###
+from node:16.13 as builder
+run mkdir -p /hubs/admin/ && cd /hubs
+copy package*.json ./
+copy admin/package*.json admin/
+run npm ci
```

Comment:
Should the `package-lock.json` checked into the repo be built with node 16.13 to prevent the following?
```
Step 5/24 : run npm ci
 ---> Running in 45dcecf84c37
npm WARN old lockfile
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile
npm WARN old lockfile This is a one-time fix-up, please be patient...
```

Reply:
yep, but we'll do it in future when we are updating hubs' dependencies, this is just an initial devops update for turkey and i'd like to minimize any disturbance to the existing codebase

Reply:
I think this might be one of the reasons the build is slow. That "one-time fix-up" will happen every time and it downloads quite a lot of metadata, which slows things down. As you noted earlier, it is much faster with an older version of node/npm.

But I agree, this is a problem to be solved on the hubs side, not a problem with this workflow, so it can be tackled at a later time.